dev/com.ibm.ws.el.3.0_fat/test-applications/TestVarargsMatching.war/src/com/ibm/ws/el30/fat/varargstest/EL30VarargsMethodMatchingServlet.java

```diff
@@ -116,6 +116,16 @@ public void testString_VarargsBird() throws Exception {
 
     }
 
+    @Test
+    public void selectMethodWithNullVarargs() throws Exception {
```

Comment:
@isaacrivriv  you're adding this to the el.3.0 FAT b/c our 3.0 implementation was already behaving this way?

Reply:
Technically, the issue is specific to Eclipse's EL API and Tomcat already had guards in place to protect against these error. I'll update the issue with my findings for this but discussed with @volosied and decided to add an additional test for verifying null varargs and this was the best place to include it in.

Reply:
Ok, thanks for the clarification. This all makes sense.